src/sensai/data_transformation.py

```diff
@@ -45,30 +51,27 @@ def _fit(self, df: pd.DataFrame):
     def _apply(self, df: pd.DataFrame) -> pd.DataFrame:
         pass
 
+    @trackDFHistory
     def apply(self, df: pd.DataFrame) -> pd.DataFrame:
         if not self.isFitted():
             raise Exception(f"Cannot apply a DataFrameTransformer which is not fitted: "
                             f"the df transformer {self.getName()} requires fitting")
-
-        inputColumns = set(df.columns)
         df = self._apply(df)
-        outputColumns = set(df.columns)
-
-        self.__removedColumns = inputColumns.difference(outputColumns)
-        self.__addedColumns = outputColumns.difference(inputColumns)
         return df
 
     def getChangeInColumnNames(self):
```

Comment:
The change in columns can be quite large, making it difficult to decipher in a string.  I also might want to use the actual list of columns from the summary for one reason or another

Reply:
What I meant was that `__str__` (or another method) could return "ColumnChange(<info>)" where <info> is, for example,
* "none" if there is no change
* "added=[...], removed=[...]" if columns are added or removed
* "reordered [new, order, of, columns]" otherwise

Reply:
I pushed a change, is this what you meant?